webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx

```diff
@@ -701,79 +616,6 @@ export default class AdvancedSettingsDisplay extends React.PureComponent<Props,
         );
     };
 
-    renderPreviewFeaturesSection = () => {
-        const serverError = this.state.serverError || null;
-        const active = this.props.activeSection === 'advancedPreviewFeatures';
-        let max = null;
-        if (active) {
-            const inputs = [];
-
-            this.state.preReleaseFeaturesKeys.forEach((key) => {
-                const feature = this.state.preReleaseFeatures[key as keyof PreReleaseFeaturesType];
-                inputs.push(
-                    <div key={'advancedPreviewFeatures_' + feature.label}>
-                        <div className='checkbox'>
-                            <label>
-                                <input
-                                    id={'advancedPreviewFeatures' + feature.label}
```

Comment:
It looks like there's a commented out reference to these settings in `e2e-tests/cypress/tests/integration/channels/accessibility/accessibility_account_settings_spec.js`. Would you mind removing that?